prow/config.yaml

```diff
@@ -5175,6 +5175,26 @@ presubmits:
         - "--upload=gs://kubernetes-jenkins/pr-logs"
 
   kubernetes-sigs/poseidon:
+  - name: ci-poseidon-e2e-gce
+    context: ci-poseidon-e2e-gce
+    interval: 24h
+    always_run: true
```

Comment:
er do you want an interval job or a postsubmit? for a postsubmit the interval field is meaningless. for a job that runs every 24 hours drop the `kubernetes-sigs/poseidon` bit, move it down below, and update the --repo flag to specify the repo without using the $env